src/middlewared/middlewared/schema.py

```diff
@@ -690,3 +691,16 @@ def to_json_schema(self, parent=None):
             schema['title'] = self.verbose
             schema['_required_'] = self.required
         return schema
+
+
+class Datetime(Attribute):
+    def validate(self, value):
+        if value is None and not self.required:
+            return self.default
+
+        if not isinstance(value, datetime):
+            # FIXME: midclt cannot supply a timeout
```

Comment:
That is not part of the schema, that comment is not suited for this class I think.

Reply:
The schema is the issue with accepting it, as we can't pass a datetime.datetime object from the CLI

Reply:
The schema can, its midclt issue.